manim/scene/scene.py

```diff
@@ -146,6 +146,9 @@ def __init__(
             random.seed(self.random_seed)
             np.random.seed(self.random_seed)
 
+    def clear():
```

Comment:
Methods need `self` as an argument, and we can also document the return type:

```suggestion
    def clear(self) -> None:
```